### ISSUE TITLE
Update background text to include instructions on how to connect via localhost

### ISSUE THREAD

**Issue by ericfont** [pelya/xserver-xsdl#59]
XSDL works great for me both via WI-FI and via localhost (DISPLAY=127.0.0.1:0) or via plain Unix sockets (DISPLAY=:0).  However, the text that is displayed does not include instructions on how to connect via localhost IP or via local Unix socket.  

Also, if I have WIFI-disabled on my android, then currently the instructions read:

<pre>Launch these commands on your Linux PC:

To tunnel X over SSH, forward port 6000 in your SSH client</pre>

I think would be better to not even include the text "Launch these commands on your Linux PC:" if WIFI is disabled.  I see from your patch https://github.com/ViewTouch/viewtouch/blob/master/android/viewtouch.patch that is where the text gets displayed.

So in summary, only display instructions for "Linux PC" if have a valid wifi ip.  But always display instructions for localhost.  

Also instructions for localhost might help users who want to use localhost, but who get confused by the text.  e.g. issue #15

**Comment by ericfont** [comment]
I was also going to suggest removing the "metacity & gimp" text, and maybe replace it with "then execute your desired X-program"

**Comment by pelya** [comment]
Somewhat fixed in 1.11.39

**Comment by ericfont** [comment]
yup, thanks!
